src/dex/camelot/camelot.ts

```diff
@@ -44,6 +44,10 @@ import { SimpleExchange } from '../simple-exchange';
 import { applyTransferFee } from '../../lib/token-transfer-fee';
 import * as CALLDATA_GAS_COST from '../../calldata-gas-cost';
 import { SolidlyData } from '../solidly/types';
+import {
+  OnPoolCreatedCallback,
+  UniswapV2Factory,
+} from '../uniswap-v2/uniswap-v2-factory';
```

Comment:
@mwamedacen Seems the issue is actual for univ2 too. Why not implement the fix for it too?

Reply:
Prioritization